AutocompleteClient/FW/Logic/Request/CIOTrackRecommendationResultsViewData.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  CIOTrackRecommendationResultsViewData.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+/**
+ Struct encapsulating the parameters that must/can be set set in order to track browse result click
+ */
+struct CIOTrackRecommendationResultsViewData: CIORequestData {
+
+    let podID: String
+    let url: String
+    let numResultsViewed: Int?
+    let resultPage: Int?
+    let resultCount: Int?
+    let sectionName: String?
+    let resultID: String?
+
+    func url(with baseURL: String) -> String {
+        return String(format: Constants.TrackRecommendationResultsView.format, baseURL)
+    }
+
+    init(podID: String, url: String, numResultsViewed: Int? = nil, resultPage: Int? = nil, resultCount: Int? = nil, sectionName: String? = nil, resultID: String? = nil) {
```

Comment:
might need to default url to "Not Available"

Reply:
Updated! 👍